styles/styles.css

```
@@ -332,6 +400,7 @@ input[placeholder] {
 
 .benefits-item-width{
   width: 400px;
+  max-width: 400px;
 }
```

лучше выбрать что-то одно, если нет резины. Ну и пробел не забывай)
```suggestion
.benefits-item-width {
  width: 400px;
}
```